math/geometry/straightlines.go

```diff
@@ -1,10 +1,11 @@
+// Package geometry contains geometric algorithms
 package geometry
 
 import (
 	"math"
 )
 
-// Defines a point with x and y coordinates.
+// Point Defines a point with x and y coordinates.
```

Comment:
```suggestion
// Point defines a point with x and y coordinates.
```
Do this for all the places you have added the name.

Reply:
Done